contracts/IFeralfileSaleDataV2.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {IFeralfileSaleData} from "./IFeralfileSaleData.sol";
+
+interface IFeralfileSaleDataV2 is IFeralfileSaleData {
+    struct SaleDataV2 {
```

Comment:
Should be `SaleData`

Reply:
Since the `SaleData` name has already been used, we need a different name for this struct.

Reply:
Ok. I thought you could use separate interface but it seems like we have to reuse the `RevenueShare`